lib/providers/providers.dart

```diff
@@ -126,4 +129,19 @@ final providers = [
     create: (_) => CreateAccountBloc(),
     update: (_, accountGeneratorService, createAccountBloc) => createAccountBloc..update(accountGeneratorService),
   ),
+  Provider(
+    create: (_) => PermissionService(),
+  ),
+  ProxyProvider<LinksService, ScannerService>(
+    create: (_) => ScannerService(),
+    update: (_, linksService, scannerService) => scannerService..update(linksService),
+  ),
+  ProxyProvider<LinksService, ScannerService>(
+    create: (_) => ScannerService(),
+    update: (_, linksService, scannerService) => scannerService..update(linksService),
+  ),
```

Comment:
This provider seems the exat same as the one above it - mistake?

Reply:
Yep, good that you caught it 👍